libkv.go

```diff
@@ -1,3 +1,64 @@
+// Package libkv provides a Go native library to store metadata.
+//
+// The goal of libkv is to abstract common store operations for multiple
```

Comment:
I also see some issues with the text "in general", but @moxiegirl might have some better ideas there (if we have the information that should be communicated/described clear)